### DISCUSSION TITLE
Node versions expected are out of date

### DISCUSSION THREAD

**Discussion by dhenson02** [uNetworking/uWebSockets.js#353] [Q&A] [answered]
build.c looks for v12.13.0 and v14.0.0 - they are now at v12.18.3 and v14.7.0.

Build fails without this change - will submit PR

**Comment by ghost** [comment]
Where does it fail? Last build is from 2 days ago and it didn't fail

**Comment by dhenson02** [comment]
It fails if you have Node 12.18.3

**Comment by ghost** [comment, marked as answer]
It doesn't fail, I just tested it in 12.18.3. The whole point of a stable ABI is that you can compile the binary with any version of the headers and it runs on all Node.js version with the same ABI.
